src/amr/data/particles/refine/split_1d.h

```diff
@@ -0,0 +1,197 @@
+/*
+Splitting reference material can be found @
+  https://github.com/PHAREHUB/PHARE/wiki/SplitPattern
+*/
+
+#ifndef PHARE_SPLIT_1D_H
+#define PHARE_SPLIT_1D_H
+
+namespace PHARE::amr
+{
+using namespace PHARE::core;
+
+template<>
+struct PinkDispatcher<DimConst<1>> : SplitPattern<DimConst<1>, RefinedParticlesConst<2>>
+{
+    using Super = SplitPattern<DimConst<1>, RefinedParticlesConst<2>>;
+
+    constexpr PinkDispatcher(float const weight, float const delta)
+        : Super{weight}
+    {
+        Super::deltas_[0] = {delta};
+        Super::deltas_[1] = {-delta};
+    }
+};
+
+
+/**************************************************************************/
+using SplitPattern_1_1_2_Dispatcher = PatternDispatcher<PinkDispatcher<DimConst<1>>>;
+
+template<>
+struct Splitter<DimConst<1>, InterpConst<1>, RefinedParticlesConst<2>>
+    : public ASplitter<DimConst<1>, InterpConst<1>, RefinedParticlesConst<2>>,
+      SplitPattern_1_1_2_Dispatcher
+{
+    constexpr Splitter()
+        : SplitPattern_1_1_2_Dispatcher{{weight[0], delta[0]}}
+    {
+    }
+
+    static constexpr std::array<float, 1> delta  = {0.551569};
```

Comment:
I do not get why the delta value should be written here, while it is written in `splitting.yml`. By the way, I like to have all the weights & deltas values written at the same place.

Reply:
I don't think it's possible to read YAML at compile time.